iOS/DuckDuckGo/MainViewController.swift

```diff
@@ -2613,6 +2613,7 @@ extension MainViewController: OmniBarDelegate {
             }
         )
 
+        DailyPixel.fireDailyAndCount(pixel: .aiChatLegacyOmnibarAichatButtonPressed)
         fireAIChatUsagePixelAndSetFeatureUsed(.openAIChatFromAddressBar)
```

Comment:
Similar to above. I’d rather use consistent pixel names and so a new pixel here.